### ISSUE TITLE
ADE20k number of classes

### ISSUE THREAD

**Issue by AAnoosheh** [CSAILVision/semantic-segmentation-pytorch#182]
So I have a question about the ADE20k itself.

I read all the seg-masks from the training set (~15k files) and counted the number of unique class values. I got 2231 unique values, where the highest value is 3144. This makes no sense as the number of classes is supposed to be 150.

I'm using this code to load the *_seg.png files in Python (adapted from the Matlab code on the dataset site):

```
mask = np.array(Image.open(mask_path), dtype=np.uint16)
R,G,B = mask[:,:,0], mask[:,:,1], mask[:,:,2]
class_mask = R // 10 * 256 + G
```

**Comment by hangzhaomit** [comment]
@AAnoosheh 
You were checking the full dataset.
In this repo, we only use 150 classes in it, which is the standard benchmark.
http://sceneparsing.csail.mit.edu/

**Comment by shamangary** [comment]
Omg I spent two days to find this solution....is it possible to have a note about this on the main page? Thank you for the great work.

**Comment by fido20160817** [comment]
where to download *_seg.png, I didn't find it in the download zip file.

**Comment by fido20160817** [comment]
I found it here https://github.com/Lextal/SotA-CV/blob/master/content/datasets/ade20k.md
